templates/401.html

```diff
@@ -0,0 +1,19 @@
+{% extends '_layout.html' %}
+
+{% block title %}401: Not Authorised{% endblock %}
+
+{% block content %}
+<div class="p-strip is-deep is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-6 u-vertically-center u-align--center">
+      <img src="https://assets.ubuntu.com/v1/bcdcf2c8-image-404.svg?w=365" alt="Error owl" width="365" />
+    </div>
+    <div class="col-6 u-vertically-center">
+      <div>
+        <h1>401: Not Authorised</h1>
```

Comment:
We use sentence case in titles (see [copy reviews](https://canonical-web-and-design.github.io/practices/content/copy-reviews.html)) meaning that this should have a small "a" on "authorised".

But maybe we should just say "Unauthorised" instead? It's shorter, and closer to the [official HTTP wording](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_errors) of "Unauthorized".